Scripts/RebuildMids.bat

```diff
@@ -0,0 +1,7 @@
+cd ..
+cd DRODUtil
+cd Release
+DRODUtil.exe import -t ../../Texts/
+cd ..
+cd ..
+git checkout Data/drod5_0.dat
```

Comment:
Is the goal here to not change the barebones .dat file (with only the message texts in it) when adding new MIDs?
It's fine to check in updates to this file along with changes to mids.h.

Reply:
I wanted to avoid pushing that file to avoid conflicts! I'll remove those lines for now.

Reply:
Sure, that's fine.  I'd add a comment to the batch file explaining this, and that this final line should be commented out to commit a new version.